src/components/CookieConsentBanner.js

```diff
@@ -99,9 +108,9 @@ const CookieConsentBanner = () => {
               <Text transparent type="secondary">
                 Allowing us to use cookies does not allow us to identify you.
                 For more information please see our{` `}
-                <a href="https://www.neontribe.co.uk/privacy-policy">
+                <InternalLink to="/privacy-policy">
```

Comment:
This removes the underline - a backwards step for not being differentiable by colour alone?

Reply:
Good catch. I had assumed it was already applied by `InternalLink` or global styles to enforce consistency.